chain/network/src/network_protocol/mod.rs

```diff
@@ -518,7 +520,6 @@ pub enum RoutedMessageBody {
     Ping(Ping),
     Pong(Pong),
     VersionedPartialEncodedChunk(PartialEncodedChunk),
-    VersionedStateResponse(StateResponseInfo),
```

Comment:
Is there a problem if I removed this field?

Reply:
Yes, there is. Removing stuff from here will break the backward compatibility of the RoutedMessageBody when it is serialised by Borsh.